spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
```

Comment:
It'd be better to include this at the config level, in either `spec_helper` or `rails_helper`.